manifests/ftpusers.pp

```diff
@@ -0,0 +1,25 @@
+# Populate /etc/ftpusers
```

Comment:
This should go into its own module and *not* be called by default.

No system has this file by default.

It should be included by `vsftpd`